qiita_db/handlers/test/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -----------------------------------------------------------------------------
```

Comment:
I know we haven't been very diligent about this but the folder's name should probably be tests instead of test ... just sayin'